### PR TITLE
PLANET-7301: Ensure to not re-instantiate the Carousel Header block from plugin

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-7301

# Description
Move the CarouselHeader block to master-theme

Related to https://github.com/greenpeace/planet4-master-theme/pull/2171

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [neptune](https://www-dev.greenpeace.org/test-neptune) | [admin](https://www-dev.greenpeace.org/test-neptune/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-neptune/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-neptune) | [composer-local.json](https://github.com/greenpeace/planet4-test-neptune/blob/main/composer-local.json)

:watch: 2023.12.12 13:40:02

File: webpack.config.js [hunk 1]
```diff
@@ -48,7 +48,6 @@ const publicJsConfig = {
     CounterScript: './assets/src/blocks/Counter/CounterScript.js',
     SplitTwoColumnsScript: './assets/src/blocks/SplitTwoColumns/SplitTwoColumnsScript.js',
     ArticlesScript: './assets/src/blocks/Articles/ArticlesScript.js',
-    CookiesScript: './assets/src/blocks/Cookies/CookiesScript.js',
```
Comment (lithrel) [inline webpack.config.js:51]: Is the Cookie scripts removal part of this PR ?

Comment (GP-Dan-Tovbein) [reply webpack.config.js:51]: Oh no! Good catch 🙏

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**